### PR TITLE
fix: only comment about no contributions after checking all contributors

### LINKED ISSUE #468: Bot both creates a PR and complains about no contributions when extra text in comment
## Describe the bug

With a(n accidentally) specially crafted comment, I've tricked the bot into both creating a PR and saying in the same comment that it couldn't determine any contributions:

> @JoshuaKGoldberg
> 
> I've put up [a pull request](https://github.com/JoshuaKGoldberg/template-typescript-node-package/pull/430) to add @JoshuaKGoldberg! 🎉
> 
> I couldn't determine any contributions to add, did you specify any contributions? Please make sure to use [valid contribution names](https://allcontributors.org/docs/en/emoji-key).

https://github.com/JoshuaKGoldberg/template-typescript-node-package/pull/429#issuecomment-1546457872

### To Reproduce

1. Set up allcontributors in a new repo
2. Post a comment with content after the `@all-contributors please add ...` line

### Expected behavior
The bot should ignore the extra content

## Additional context

I posted some more comments on that PR. It looks like the bug might have something to do with text like `all-contributors` existing?

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
**What**: Fixes #468.

**Why**: Currently, the bot comments _"I couldn't determine ..."_ as soon as a potential contributor is found to not have any contributions. But if there are multiple potential contributors, that comment shouldn't happen.

**How**: With these changes, the comment is only added after all potential contributors are checked for sending a PR.

**Checklist**:

- [x] Documentation _(n/a for bug fix)_
- [x] Ready to be merged 
- ~[ ] Added myself to contributors table.~

### CODE REVIEW

File: test/fixtures/issue_comment.created.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "created_at": "2019-01-10T08:36:52Z",
     "updated_at": "2019-01-10T08:36:52Z",
     "author_association": "MEMBER",
-    "body": "@all-contributors please add jakebolam for code, doc and infra"
+    "body": "@all-contributors please add jakebolam for code, doc and infra\n🤖 Beep boop! This comment was added automatically by [all-contributors-auto-action](https://github.com/marketplace/actions/all-contributors-auto-action).\nNot all contributions can be detected from Git & GitHub alone. Please comment any missing contribution types this bot missed.\n...and of course, thank you for contributing! 💙"
```
Comment (gr2m) [inline test/fixtures/issue_comment.created.json:82]: I don't quite get this change, that does not look like a comment that a user would post?

Comment (JoshuaKGoldberg) [reply test/fixtures/issue_comment.created.json:82]: It's a comment a bot of mine posts that hits the issue. 🙂 

Example: https://github.com/JoshuaKGoldberg/create-typescript-app/pull/429#issuecomment-1546457807

Comment (gr2m) [reply test/fixtures/issue_comment.created.json:82]: ohhh never seen this action! Cool!

**Comment by JoshuaKGoldberg** [comment]
Hooray! Thanks @gr2m!

**Pull request merged**